std_msgs/Header header

float32 accelerationLateralErrAmp
uint8 accelerationLateralErrAmpInvalidFlag
uint8 qualifierAccelerationLateral 
float32 accelerationLateral
uint8 accelerationLateralInvalidFlag
uint8 accelerationLateralEventDataQualifier

